src/test/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/utils/TemplateHelpersTest.kt

```diff
@@ -265,4 +265,19 @@ class TemplateHelpersTest {
       assertThat(response).isEqualTo(expectedValue)
     }
   }
+
+  @Nested
+  inner class EqualsTest {
+    @Test
+    fun `eq returns true when args are equal`() {
+      val response = templateHelpers.eq("Value 1", "Value 1")
+      assertThat(response).isTrue()
+    }
+
+    @Test
+    fun `eq returns false when args are equal`() {
```

Comment:
should this title be "eq returns false when args are **not** equal"

do we need any tests for the null cases? are we not able to use the standard `.equals()` in mustache?